nomination/url_handler.py

```diff
@@ -172,17 +179,27 @@ def check_url(url):
 def get_nominator(form_data):
     try:
         # Try to retrieve the nominator
-        nominator = Nominator.objects.get(nominator_email=form_data['nominator_email'])
-    except (Nominator.DoesNotExist, Nominator.MultipleObjectsReturned):
+        nominator, created = Nominator.objects.get_or_create(
+                             nominator_email=form_data['nominator_email'],
+                             defaults={
+                                       'nominator_name': form_data['nominator_name'],
+                                       'nominator_institution': form_data['nominator_institution']
+                                      })
+
+    except Nominator.MultipleObjectsReturned:
+        # Retrieve unique nominator
         try:
-            # Create a new nominator object
-            nominator = Nominator(nominator_email=form_data['nominator_email'],
-                                  nominator_name=form_data['nominator_name'],
-                                  nominator_institution=form_data['nominator_institution'])
-            nominator.save()
-        except Exception:
+            nominator = Nominator.objects.get(
```

Comment:
So we get here if there were multiple objects with the nominator's email. That's fine, but what happens if there is _no_ existing object with the exact combination of email, name, and institution?

Reply:
since we are using `get_or_create` in try, if an object doesn't exist already, it'll be created.

Reply:
That is in the try branch, but if we get _here_ it is because multiple nominators with the same email address were returned. So I'm wondering what happens if there is no nominator among those multiple ones returned that has the specified combination of name and institution. I believe in that case we would be returning an empty nominator, when maybe (I'm just guessing, don't know for sure) we would instead like to return `False` as in the except block on line 197?

Reply:
I think I can return False as you suggested, as we are returning 'False' here to give a message to the user to contact admin when they have multiple accounts but aren't able to retrieve due to existance of multiple accounts with same name, email, institution or the case you raised about where the mentioned institution or name do not exist.

Reply:
Do you all think we should drop the  `Nominator.MultipleObjectsReturned` handling, since we are now making the `nominator_email` unique? or keep it in case someone is already using this app, has duplicates, and doesn't run the migration for some reason?

Reply:
I think we can keep it to handle already existing duplicates.